kms/decrypt_asymmetric.go

```diff
@@ -37,17 +39,39 @@ func decryptAsymmetric(w io.Writer, name string, ciphertext []byte) error {
 		return fmt.Errorf("failed to create kms client: %v", err)
 	}
 
+	// Optional but recommended: Compute ciphertext's CRC32C.
+	crc32c := func(data []byte) int64 {
+		t := crc32.MakeTable(crc32.Castagnoli)
+		// Convert to int64 to match API's use of Int64Value wrapper for CRC32C fields
```

Comment:
I'm not sure we need this comment? And it might be better to handle the int64 conversion when building the request, rather than in this helper function?

Reply:
sg. will address in a followup commit.

Reply:
Then again, it seems that even for response verification the casting is required. To minimize casting-related clutter, I will cast once in the crc32c() function. 

See fails: https://source.cloud.google.com/results/invocations/19166235-a7aa-4377-9cd6-71767514ead5/targets/cloud-devrel%2Fgo%2Fgolang-samples%2Fpresubmit/log

Reply:
Never mind. It is clearer when done at the call site of crc32c, I think. :)